go/filemanager/pfsmodules/command.go

```diff
@@ -33,13 +34,16 @@ type Command interface {
 
 // CheckUser checks if a user has authority to access a path.
 // path example:/pfs/$datacenter/home/$user
-func checkUser(path string, user string) error {
-	a := strings.Split(path, "/")
-	if len(a) < 3 {
+func checkUser(pathStr string, user string) error {
+	pathStr = path.Clean(strings.TrimSpace(pathStr))
+	a := strings.Split(pathStr, "/")
```

Comment:
Should use `os.PathSeparator` instead of "/" to support Windows, or use `filepath.Split()`